### DISCUSSION TITLE
Monitoring using Maester for Configuration drift, get only emails when configuration drifts

### DISCUSSION THREAD

**Discussion by sergioandreslq** [maester365/maester#449] [Q&A]
Hello team, 
I saw the activity to activate monitoring using GitHub Actions and the email send after run the maester assessment,

the idea is to receive only the email if there is configuration drift in the tenant, do you know if maester is able to compare configuration from yesterday and configuration from today, then, if there is configuration drift notify by email?

best regards,

**Comment by spaelling** [comment]
It also produces a json file as output. You could make a comparison with previous ones to identify configuration drift
